### PR TITLE
DB-10259: fix import for varchar bit column

### PR DESCRIPTION
**Pull request by jyuanca**

### CODE REVIEW

**Comment by jyuanca** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by yxia92 — APPROVED** [review]
+1

File: db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLBit.java [hunk 1]
```diff
@@ -229,6 +230,30 @@ else if (sourceWidth > desiredWidth)
...
+				if (SanityManager.DEBUG)
+					SanityManager.THROWASSERT(
+							"value is null, stream or blob is not null");
+				return "";
+			}
+		}
+		else
+		{
+			String hex = com.splicemachine.db.iapi.util.StringUtil.toHexString(dataValue, 0, dataValue.length);
+			return String.format("X'%s'",hex);
```
Comment (yxia92) [inline db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLBit.java:252]: It seems that the only difference between the toString() function in SQLBit and that in SQLBinary is the call to String.format(), why don't we move this implementation up to SQLBinary instead? It seems to make sense to apply String.format() there too.

**Review by yxia92 — CHANGES_REQUESTED** [review]
I have a question posted earlier.

**Review by dgomezferro — CHANGES_REQUESTED** [review]
Marking as changes requested until @yxia92 's comment is resolved

File: db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLBit.java (hunk 1 above)
Comment (jyuanca) [reply db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLBit.java:252]: This impacts all SQLBinary's subclasses, include SQLBlob. X'whatever' works for char/varchar/ long char bit data, not for Blob

Comment (yxia92) [reply db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLBit.java:252]: Ok, thanks for the explanation!

**Review by yxia92 — APPROVED** [review]

**Review by dgomezferro — APPROVED** [review]

**Pull request merged**